.github/workflows/terraform_lint.yaml

```diff
@@ -5,21 +5,22 @@ on:
 
 jobs:
   tf_lint:
-    name: 'Terraform lint'
+    name: 'Terraform lint (on tf ${{ matrix.terraform_version }})'
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        terraform_version: [ '~1.5.0', '~1.6.0', '~1.7.0', 'latest'] # in theory, we go down to 1.0, but I think that's overkill for lint
```

Comment:
`latest` should work here; and will give us a deterministic name to use in the status check